bin/utils/imagehandler.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+import cv2
+import logging
+
+
+class ImageHandler:
+    """
+    Constructor to create an image handler
+    Handles file operations over images
+    """
+
+    def loadImage(self, image_name):
```

Comment:
In `getconfig you were using `snake_case`  for method names, but here you are using `CamelCase` please stick to a single naming standard